src/app/components/client/ProgressCard.tsx

```diff
@@ -0,0 +1,110 @@
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ "use client";
+
+ import { CSSProperties, useState } from "react";
+ import { QuestionMarkCircle } from "../server/Icons";
+ import styles from "./ProgressCard.module.scss";
+ import ExploringLaptopPlus from "./assets/exploring-laptop-check.png";
+ import ExploringLaptopMinus from "./assets/exploring-laptop-minus.png";
+ import SparklingCheck from "./assets/sparkling-check.png";
+ import Image from "next/image";
+import { Modal } from "./Modal";
+ 
+ export type Props = {
+   resolvedByYou: number;
+   autoRemoved: number;
+   totalNumExposures: number;
+ };
+ 
+ function PercentageComplete(props: Props) {
+   const percentageCompleteNum =
+     ((props.autoRemoved + props.resolvedByYou) / props.totalNumExposures) * 100;
+   return percentageCompleteNum;
+ }
+ 
+ export const ProgressCard = (props: Props) => {
+   const percentageCompleteNum = PercentageComplete(props);
+   const percentageRemainingNumber = 100 - PercentageComplete(props);
+
+   const [isModalOpen, setIsModalOpen] = useState(false);
+
+   const openModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const closeModal = () => {
+    setIsModalOpen(false);
+  };
+
+   const activeProgressBarStyle: CSSProperties = {
+     width: `${percentageCompleteNum}%`,
+   };
+ 
+   const ProgressBar = () => {
+     return (
+       <div className={styles.progressBarContainer}>
+        <div className={styles.fullProgressBar}>
+          <div
+            className={styles.activeProgressBar}
+            style={activeProgressBarStyle}
+          ></div>
+           <div className={styles.percentageBreakdown}>
+            <p>{percentageCompleteNum}% complete</p>
+            <p>{percentageRemainingNumber}% in progress</p>
+          </div>
+        </div>
+         <Image src={SparklingCheck} alt="" />
+       </div>
+     );
+   };
+
+   const modalContent = (
+    <>
+    <p>
+      <strong>Resolved by you</strong> includes anything you have manually fixed. 
+      All data breaches that require access to your accounts need to 
+      be fixed manually, even if you have upgraded to Premium. 
+      <br/><br/>
+      <strong>Auto-removed</strong> includes any exposures from data broker profiles that 
+      we have removed for you. This is available only for Premium subscribers.
+      Complete includes anything resolved by you or auto-removed by us. 
+      <br/><br/>
+      <strong>In Progress</strong> includes anything that we are currently working on fixing. 
+      Removals typically take 7-14 days but the most difficult sites could take 
+      longer. You may also start to see removals happening within the same day.
+       </p>
+    </>
+   )
+ 
+   return (
+     <div className={styles.progressCard}>
+       <div className={styles.header}>
+         Here is what we fixed
+         <button onClick={openModal}>
+           <QuestionMarkCircle alt={""} width="15" height="15" />
```

Comment:
Nit: The curly braces around the quotes are not needed.
```suggestion
           <QuestionMarkCircle alt="" width="15" height="15" />
```